### PR TITLE
Add approval comment text

### PR DESCRIPTION
**Pull request by tapishr**
Please check the action items covered in the PR - 

- [x] Build is running
- [ ] Eventing is functional and tested
- [ ] Unit or integration tests added and running
- [ ] Manual QA

### CODE REVIEW

**Review by MuskanPaliwal — APPROVED** [review]

**Review by avikalpg — APPROVED** [review]

File: vibi-dpu/src/core/approval.rs [hunk 1]
```diff
@@ -24,7 +25,21 @@ pub async fn process_approval(deserialised_msg_data: &Value) {
...
+    let mut coverage_map_obj = CoverageMap::new(repo_provider);
+    coverage_map_obj.calculate_coverage_map(relevance_vec, reviewer_handles);
     // add up contribution of aliases
     // add comment
+    let comment_text = approval_comment_text(&coverage_map_obj);
+    // get access token and call add_comment in gh/bb
+}
+
+fn approval_comment_text(coverage_map: &CoverageMap) -> String {
+    let mut comment = "Relevant users for this PR:\n\n".to_string();  // Added two newlines
```
Comment (avikalpg) [inline vibi-dpu/src/core/approval.rs:37]: We should ideally start by telling the right relevant reviewer approval percentage of the PR.
Let me know if that value is not available to us here.

**Review by avikalpg — DISMISSED** [review]
Approved because the approval comment contents can be improved later.

File: vibi-dpu/src/core/approval.rs (hunk 1 above)
Comment (tapishr) [reply vibi-dpu/src/core/approval.rs:37]: Sounds correct. Can do it

File: vibi-dpu/src/utils/coverage.rs [hunk 2]
```diff
@@ -113,4 +126,28 @@ impl CoverageMap {
             self.provider.clone(), handle.to_owned(), relevance);
         self.handle_map.insert(handle.to_owned(), coverage);
     }
+
+    pub fn generate_coverage_table(&self) -> String {
+        let mut comment = "| Contributor Name/Alias  | Relevance |\n".to_string();  // Added a newline at the end
+        comment += "| -------------- | --------------- |\n";  // Added a newline at the end
+        let mut rev_handles: Vec<_> = self.handle_map.keys().filter(|k| !k.is_empty()).cloned().collect();
+        rev_handles.sort();
+        for handle in rev_handles {
```
Comment (tapishr) [inline vibi-dpu/src/utils/coverage.rs:135]: Get same table as relevance with this - :white_check_mark:

File: vibi-dpu/src/utils/coverage.rs [hunk 3]
```diff
@@ -113,4 +126,28 @@ impl CoverageMap {
             self.provider.clone(), handle.to_owned(), relevance);
         self.handle_map.insert(handle.to_owned(), coverage);
     }
+
+    pub fn generate_coverage_table(&self) -> String {
+        let mut comment = "| Contributor Name/Alias  | Relevance |\n".to_string();  // Added a newline at the end
+        comment += "| -------------- | --------------- |\n";  // Added a newline at the end
+        let mut rev_handles: Vec<_> = self.handle_map.keys().filter(|k| !k.is_empty()).cloned().collect();
```
Comment (tapishr) [inline vibi-dpu/src/utils/coverage.rs:133]: highest percentage relevance first

**Pull request merged**